src/develop/blend.c

```diff
@@ -673,7 +673,7 @@ static void _refine_with_detail_mask_cl(struct dt_iop_module_t *self, struct dt_
                                 const struct dt_iop_roi_t *roi_out, const float level, const int devid)
 {
   if(level == 0.0f) return;
-  const gboolean info = ((darktable.unmuted & DT_DEBUG_MASKS) && (piece->pipe->type == DT_DEV_PIXELPIPE_FULL));
```

Comment:
The new version seems more correct, right?

We had : check that the type IS DT_DEV_PIXELPIPE_FULL only.

The new version is : check that that we are asking for DT_DEV_PIXELPIPE_FULL even if some other bits are set.

Reply:
exactly, this way we also do it in fast mode and we should really have that.